src/lib/routes/admin-api/tag-type.ts

```diff
@@ -69,8 +77,11 @@ class TagTypeController extends Controller {
                 openApiService.validPath({
                     tags: ['Tags'],
                     operationId: 'createTagType',
+                    summary: 'Create a tag type',
+                    description: 'Create a new tag type.',
```

Comment:
What happens if a tag type with the same name already exists? Is that a 409? We should communicate that to the end user.

Reply:
Added here: [bf74c88](https://github.com/Unleash/unleash/pull/4194/commits/bf74c88ce74fd68414cacd8cc89a0c33f7e83ad3)